cmd/training-operator.v1/main.go

```diff
@@ -19,6 +19,7 @@ package main
 import (
 	"flag"
 	"fmt"
+	"go.uber.org/zap/zapcore"
```

Comment:
Please move this import statement to the second group.

Reply:
ok,moved